README.md

```diff
@@ -5,7 +5,8 @@
 ### Building:
 
 ```bash
-git clone https://github.com/opencontainers/runc
+git clone https://github.com/opencontainers/runc $GOPATH/src/github.com/opencontainers/runc
```

Comment:
@carmark `go get -d github.com/opencontainers/runc` and `git clone https://github.com/opencontainers/runc $GOPATH/src/github.com/opencontainers/runc` are roughly equivalent, except `go get` correctly handles `GOPATH`. We should avoid path munging in this build file.

Please carefully read [Writing Go Code](https://golang.org/doc/code.html) for more information on correctly setting up your development environment and managing `GOPATH`. Most build problems with Go stem from not following those concepts carefully.